app/controllers/ideas_controller.rb

```diff
@@ -73,6 +74,20 @@ def set_idea
     @idea = Idea.find(params[:id] || params[:idea_id])
   end
 
+  def update_review_date(idea_params)
```

Comment:
I think the controller is not the right place to do this work. It's probably best to do this on the model (or in a service class).